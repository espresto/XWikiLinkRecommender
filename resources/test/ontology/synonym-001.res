[]
[http://www.ag-csw.de/ontologies/gewuerz#Sommerzwiebel, http://www.ag-csw.de/ontologies/gewuerz#Hauszwiebel]
