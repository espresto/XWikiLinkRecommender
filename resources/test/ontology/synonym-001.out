[]
[http://www.ag-csw.de/ontologies/gewuerz#Sommerzwiebel]
